build_release_vs2022.bat

```diff
@@ -56,9 +57,10 @@ cd %WP%
 mkdir %build_dir%
 cd %build_dir%
 
-echo cmake .. -G "Visual Studio 17 2022" -A x64 -DBBL_RELEASE_TO_PUBLIC=1 -DCMAKE_PREFIX_PATH="%DEPS%/usr/local" -DCMAKE_INSTALL_PREFIX="./OrcaSlicer" -DCMAKE_BUILD_TYPE=%build_type%
-cmake .. -G "Visual Studio 17 2022" -A x64 -DBBL_RELEASE_TO_PUBLIC=1 -DCMAKE_PREFIX_PATH="%DEPS%/usr/local" -DCMAKE_INSTALL_PREFIX="./OrcaSlicer" -DCMAKE_BUILD_TYPE=%build_type% -DWIN10SDK_PATH="C:/Program Files (x86)/Windows Kits/10/Include/10.0.22000.0"
+echo on
+cmake .. -G "Visual Studio 17 2022" -A x64 -DBBL_RELEASE_TO_PUBLIC=1 -DCMAKE_PREFIX_PATH="%DEPS%/usr/local" -DCMAKE_INSTALL_PREFIX="./OrcaSlicer" -DCMAKE_BUILD_TYPE=%build_type% -DWIN10SDK_PATH="%WindowsSdkDir%Include\%WindowsSDKVersion%\"
```

Comment:
Where is the Windows SDK path/version being gotten from? If you are pulling it from the ENV variables set by CMake for the deps build, remember that the slicer can be build independently of the deps and vice versa

Reply:
It's from x64 Native tool window environment, so it's set before BAT file is run.

Reply:
Only problem is if you don't run from that environment. I typically run that script from the CLion terminal which results in the path not being set correctly. Maybe check if the variables are initialized and if not set the current values as default?

Reply:
Actually build instruction clearly says `Run build_release.bat in x64 Native Tools Command Prompt for VS 2019` (applied to 2022 as well I guess).